elasticdl/doc/tutorials/elasticdl_local.md

```diff
@@ -0,0 +1,112 @@
+# ElasticDL on local
+
+This document aims to give a simple example to show how to sumbmit deep learning jobs to a local kubernetes cluster in a local computer. It helps to understand the working process of ElasticDL.
+
+
+## Environment prepare
+
+Here, we use macbook as our expriment environment.
+
+
+1. Install docker
+
+```bash
+brew cask install docker
+```
+
+2. Install kunernetes
+
+```bash
+brew install kubectl
+brew cask install minikube
+brew install docker-machine-driver-hyperkit
+```
+
+
+And change permission of hyperkit:
+
+```bash
+sudo chown root:wheel /usr/local/opt/docker-machine-driver-hyperkit/bin/docker-machine-driver-hyperkit
+sudo chmod u+s /usr/local/opt/docker-machine-driver-hyperkit/bin/docker-machine-driver-hyperkit
+```
+
+## Write model file
+
+We use Tensorflow Keras API to build our models. For details, please refer to [ModelBuilding](https://github.com/sql-machine-learning/elasticdl/blob/develop/elasticdl/doc/model_building.md) part.
+
+
+## Summit Job to minikube
+
+### Install ElasticDL
+
+```bash
+git clone https://github.com/sql-machine-learning/elasticdl.git
+cd elasticdl
+python setup.py install
+```
+
+### Setup kubernetes related environment
+
+```bash
+kubectl apply -f manifests/examples/elasticdl-rbac.yaml
+minikube start --vm-driver=hyperkit --cpus 2 --memory 6144
+eval $(minikube docker-env)
+bash elasticdl/docker/build_all.sh
+```
+
+### Summit a training job
+
+
+You have to set right docker environment. Please run `minikube docker-env` to get docker host url and docker cert path.
```

Comment:
"You have to set right docker environment" -> "There are other docker settings that you might also want to configure prior to submitting the training job"

Reply:
Done